src/_P097_Esp32Touch.ino

```diff
@@ -168,17 +162,27 @@ boolean Plugin_097(uint8_t function, struct EventStruct *event, String& string)
           if (touched) {
             bitClear(p097_pinTouched, t);
             UserVar.setFloat(event->TaskIndex, 0, touchRead(CONFIG_PIN1));
+            
             if (touchInterruptGetLastStatus(CONFIG_PIN1)) {
-                Scheduler.schedule_task_device_timer(event->TaskIndex, millis());
                 UserVar.setFloat(event->TaskIndex, 1, 1);
-            } else {
-                if (P097_SEND_RELEASE_EVENT) {
-                  Scheduler.schedule_task_device_timer(event->TaskIndex, millis());
+                eventQueue.add(event->TaskIndex, F(PLUGIN_VALUENAME2_097), 1);
```

Comment:
Hmm, the user can change the Values names, so using the predefined value might not work out as expected... (there's a function to fetch the current name, but I can't recall that from memory where I'm now)